examples/distributedLock/README.md

```diff
@@ -0,0 +1,87 @@
+# Example - Distributed Lock APIs
+
+This example demonstrates the Distributed Lock APIs in Dapr.
+It demonstrates the following APIs:
+- **Distributed Lock**: Try Lock
+- **Distributed Lock**: Unlock
+
+> **Note:** Make sure to use the latest proto bindings by running scripts/fetch-proto.sh file.
+## Prerequisites
+
+- [Dapr CLI](https://docs.dapr.io/getting-started/install-dapr-cli/)
+- [Dapr JS SDK](https://docs.dapr.io/developing-applications/sdks/js/)
+- Initialize Dapr environment by pulling the code from master branch of [Dapr](https://github.com/dapr/dapr)
+
+## Overview
+This example shows the usage of two different Distributed Lock APIs, TryLock call and Unlock call.
```

Comment:
Redundant to the intro? Maybe write: "The TryLock and Unlock calls are implemented under the `client.lock` attribute" ?